scripts/globals/abilities/allies_roll.lua

```diff
@@ -48,12 +49,36 @@ function onUseAbility(caster,target,ability,action)
 end;
 
 function applyRoll(caster,target,ability,action,total)
-    local duration = 300 + caster:getMerit(MERIT_WINNING_STREAK)
+    local duration = 300 + caster:getMerit(MERIT_WINNING_STREAK) + caster:getMod(MOD_PHANTOM_DURATION)
     local effectpowers = {2, 3, 20, 5, 7, 9, 11, 13, 15, 1, 25, -5}
     local effectpower = effectpowers[total];
---    if (caster:getLocalVar("corsairRollBonus") == 1 and total < 12) then -- TODO Add AF3
---        effectpower = effectpower + 15 --  TODO Add Logic for Phantom Roll+
---    end
+-- Apply Buffs from Allies Roll Enhancing Gear if present
+    local equippedEnh = caster:getEquipID(SLOT_HANDS);
+    if (equippedEnh == 11220) then -- Navarch's Gants +1
```

Comment:
move these equip checks to a new MOD

it might seem kind of silly for only one set of items for one roll.. but there's no guarantee they won't add another item (or upgrade the AF again)

Reply:
I opted to go with the two tier logic since the +1 hands only have a 50% chance of applying the buff, but can definitely create MODs instead. The buff value appears to be uniform for AF3+2 and Reforged pieces. If I go the MOD route, my initial thought is to apply a value of 1 for AF3+1 and a value of 2 for AF3+2/Reforged. 

It would trim out a few lines of code across 5 rolls.

Reply:
or a value of 50 for AF3+1 and a value of 100 for AF3+2/119